rs/nns/governance/src/neuron/types.rs

```diff
@@ -500,20 +522,28 @@ impl Neuron {
         if topic == Topic::ExchangeRate {
             return;
         }
+
```

Comment:
Note for reviewer: The way this method works should be the same in effect as how StableNeuronStore.register_recent_neuron_ballot works, except that directly deals with the StableBTreeMap records.